mlflow/johnsnowlabs.py

```diff
@@ -0,0 +1,786 @@
+"""
+The ``mlflow.johnsnowlabs`` module provides an API for logging and loading Spark NLP and NLU models. This module
+exports the following flavors:
+
+The following Environment Variables must be present
+- `SECRET`: The secret for the John Snow Labs Enterprise NLP Library
+- `SPARK_NLP_LICENSE`: Your John Snow Labs Enterprise NLP License
+- `AWS_ACCESS_KEY_ID`: Your AWS Secret ID for accessing John Snow Labs Enterprise Models
+- `AWS_SECRET_ACCESS_KEY`: Your AWS Secret key for accessing John Snow Labs Enterprise Models
+
+
+You can set them like this in Python
+
+```python
+import os
+os.environ['SECRET'] = 'MY_SECRET'
+os.environ['AWS_ACCESS_KEY_ID'] = 'MY_AWS_ACCESS_KEY_ID'
+os.environ['AWS_SECRET_ACCESS_KEY'] = 'MY_AWS_SECRET_ACCESS_KEY'
+os.environ['SPARK_NLP_LICENSE'] = 'MY_SPARK_NLP_LICENSE'
+
```

Comment:
````uggestion
```
````

Reply:
resolved in https://github.com/mlflow/mlflow/pull/8556/commits/76d55dcdd647b6d183e933e3f7802995c95fc2fc